Tests.Integration.Presentation.Web/Users/V2/UserInternalApiV2Test.cs

```diff
@@ -68,14 +69,23 @@ public async Task Can_Update_User()
         {
             //Arrange
             var organization = await CreateOrganizationAsync();
-            var user = await CreateUserAsync(organization.Uuid); ;
+            var (_, _, token)= await HttpApi.CreateUserAndGetToken(CreateEmail(), OrganizationRole.GlobalAdmin, organization.Id, true);
```

Comment:
I don't like creating the user twice, but it's a necessary evil since we need the token, but also want to double test the V2 create endpoint